### PR TITLE
alert logic app doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/security-threat-detection/alert-logic.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+description: The Sumo Logic app for Alert Logic provides enhanced security operations by seamlessly combining the capabilities of Alert Logic's incident detection and response with Sumo Logic's advanced log management and analytics platform.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('/img/integrations/security-threat-detection/alert-logic-logo.png')} alt="thumbnail icon" width="100"/>
+
+The Sumo Logic app for Alert Logic provides enhanced security operations by seamlessly combining the capabilities of Alert Logic's incident detection and response with Sumo Logic's advanced log management and analytics platform. This powerful integration allows you to gain comprehensive insights into your security landscape, detect potential threats, and respond effectively to security incidents.
+
+Key features of the Alert Logic app include:
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/alert-logic.md:14]: @JV0812, you might want to make all the bullet items sentence case. (For example, change "Real-time Threat Visibility" to "Real-time threat visibility".) But it's not essential.

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812! Approved.

File: docs/integrations/security-threat-detection/alert-logic.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- **Automation and Orchestration**. Build automated workflows and triggers based on Alert Logic incident data to streamline incident response processes.
+- **Enhanced Security Insights**. Gain deeper insights into your security landscape by analyzing Alert Logic incidents to elevate your security posture, accelerate incident response, and effectively protect your organization against evolving threats
+- **Rapid Incident Detection**. Leverage real-time correlation and alerts, to quickly detect and respond to potential security threats.
+- **Improved Incident Response**. Empower your security teams to respond effectively to incidents by providing them with a holistic view of threat data and streamlined collaboration tools.
+- **Customizable Monitoring**. Tailor the Alert Logic dashboard to your organization's security requirements to monitor the most relevant threat vectors.
+- **Scalable Threat Hunting**. Scale your threat-hunting efforts by leveraging Sumo Logic's powerful search capabilities to hunt for threats across large volumes of data.
+
```
Comment (jpipkin1) [inline docs/integrations/security-threat-detection/alert-logic.md:27]: @JV0812, you might want to make all the bullet items in this list sentence case. (For example, change "Real-time Threat Visibility" to "Real-time threat visibility".) But it's not essential.

**Review by kimsauce — APPROVED** [review]
Hi @JV0812 , I've swapped out the image with a hi-res one

**Pull request merged**